docs/content/v2025.1/yugabyte-platform/security/create-kms-config/ciphertrust-kms.md

```diff
@@ -0,0 +1,128 @@
+      AWS
+    </a>
+  </li>
+  <li >
+    <a href="../google-kms/" class="nav-link">
+      <i class="fa-brands fa-google" aria-hidden="true"></i>
+      Google
+    </a>
+  </li>
+
+  <li >
+    <a href="../azure-kms/" class="nav-link">
+      <i class="icon-azure" aria-hidden="true"></i>
+      Azure
+    </a>
+  </li>
+
+  <li >
+    <a href="../hashicorp-kms/" class="nav-link">
+      HashiCorp
+    </a>
+  </li>
+
+  <li >
+    <a href="../ciphertrust-kms/" class="nav-link active">
+      CipherTrust
+    </a>
+  </li>
+
+</ul>
+
+Encryption at rest in YugabyteDB Anywhere supports the use of [CipherTrust KMS](https://thalesdocs.com/ctp/cm/latest/).
+
+{{<tags/feature/ea idea="1227">}}CipherTrust support is Early Access. To enable the feature in YugabyteDB Anywhere, set the **Allow CipherTrust KMS** Global Runtime Configuration option (config key `yb.kms.allow_ciphertrust`) to true. Refer to [Manage runtime configuration settings](../../../administer-yugabyte-platform/manage-runtime-config/). Note that only a Super Admin user can modify Global configuration settings.
+
+## CipherTrust Manager prerequisites
+
+You must have a running CipherTrust Manager installation, and it must be accessible to YugabyteDB Anywhere.
+
+To create a KMS configuration, you need either the credenials of a CipherTrust user, or an authentication token.
+
+- You can use a normal local user, or an LDAP-based user.
+
+  The user or group that your user belongs to requires the following Key Access Properties:
+  - Read
+  - Use
+  - Encrypt
+  - Decypt
+
+  For more information on managing users, refer to the [CipherTrust Administration](https://thalesdocs.com/ctp/cm/latest/admin/cm_admin/authentication/users/index.html) documentation.
+
+- If you use refresh tokens to authenticate, when the token expires you need to manually update the KMS configuration with the new token. For information on using authentication tokens, refer to [Authentication tokens](https://thalesdocs.com/ctp/cm/latest/admin/cm_admin/authentication/tokens/index.html#refresh-tokens) in the CipherTrust documentation.
+
+To use a pre-existing CipherTrust key, the key must have the following properties:
+
+- It must be Active.
+- Have the following Key Usage Permissions:
+  - Encrypt
+  - Decrypt
+
+YugabyteDB Anywhere can also create a CipherTrust key with the correct settings when creating a KMS configuration.
+
+## Create a KMS configuration
+
+You can create a KMS configuration that uses CipherTrust, as follows:
+
+1. Navigate to **Integrations > Security > Encryption At Rest** to access the list of existing configurations.
+
+1. Click **Create New Config**.
+
+1. Enter the following configuration details in the form:
+
+    - **Configuration Name** — Enter a meaningful name for your configuration.
+    - **KMS Provider** — Select **CipherTrust KMS**.
+    - **CipherTrust Manager URL** — Enter the URL of your CipherTrust Manager deployment.
```

Comment:
We can also add something like: "This URL must be accessible to the YBA instance"